client/components/Styles.jsx

```diff
@@ -0,0 +1,249 @@
+  font-weight:250;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  `;
+
+export const SmallIcon = styled.svg`
+  width: 14px;
+  box-sizing: border-box;
+  display: inline;
+  `;
+
+export const InfoText = styled.span`
+  margin-left: 8px;
+  display: inline;
+  `;
+
+export const StarIcon = styled.svg`
+  width: 14px;
+`;
+
+export const StarSet = styled.div`
+  display:inline;
+  margin-left: 4px;
+  margin-right: 4px;
+  vertical-align: -2px;
+  `;
+
+export const RatingInternalWrapper = styled.div`
+  display:inline;
+  font-size:14px;
+  margin-left: 4px;
+  color:#FFC48C;
+  font-family:"sf pro display", -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  `;
+
+export const Total = styled.div`
+  color:white;
+  display:inline;
+  margin-left: 4px;
+  margin-right: 4px;
+  font-weight: 300;
+  `;
+
+export const WishlistButton = styled.button`
+  position: relative;
+  color: white;
+  height: 40px;
+  width: 20%;
+  border: 1px solid white;
+  font-weight: 700;
+  background: black;
+  text-align: center;
+  align-items: center;
+  border-radius: 4px;
+  display: inline-flex;
+  min-width: 10rem;
+  padding: 0 1.2rem;
+  `;
+
+  export const ShareButton = styled.button`
+  position: relative;
+  color: white;
+  height: 40px;
+  width: 20%;
+  border: 1px solid white;
+  font-weight: 700;
+  background: black;
+  text-align: center;
+  align-items: center;
+  border-radius: 4px;
+  display: inline-flex;
+  min-width: 10rem;
+  padding: 0 1.2rem;
+  `;
+
+export const WishContents = styled.div`
+  height: 40px;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  box-sizing: border-box;
+  display: flex;
+  `;
+
+export const ShareContents = styled.div`
+  height: 40px;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  box-sizing: border-box;
+  display: flex;
+  `;
+
+export const WishAdd = styled.div`
+  display: flex;
+  margin-right: 10px;
+  `;
+
+export const ShareText = styled.div`
+  display: flex;
+  margin-right: 10px;
+  `;
+
+export const HeartIcon = styled.svg`
+  box-sizing: border-box;
+  color: white;
+  width: 24px;
+  height: 24px;
+  display: inline;
+  `;
+
+export const ShareArrow = styled.svg`
+  box-sizing: border-box;
+  width: 24px;
+  height: 24px;
+  display: inline;
+  `;
+
+export const Subject = styled.div`
+  font-family: "sf pro display", -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  color: #8ED1DC;
+  display:inline;
+`;
+
+export const Pointer = styled.div`
+  color: white;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-size: 10px;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  display:inline;
+`;
+
+export const IconNext = styled.svg`
+  fill: white;
+  width: 14px;
+`;
+
+export const Arrow = (
```

Comment:
These svg icons may be better suited in a separate file containing just the svgs